v2/internal/pkg/release/cincinnati.go

```diff
@@ -122,13 +126,39 @@ func (o *CincinnatiSchema) GetReleaseReferenceImages(ctx context.Context) ([]v2a
 	// before making a deep copy
 	// check that the "platform.release" field is not empty
 	if len(o.Config.Mirror.Platform.Release) > 0 {
-		copyImage := v2alpha1.CopyImageSchema{
-			Source:      o.Config.Mirror.Platform.Release,
-			Destination: "",
-			Origin:      o.Config.Mirror.Platform.Release,
+		// OCPBUGS-50617
+		// include signature verify and download for ga releases, rc (release candidate)
+		// and ec Iengineering candidate) by tag or digest
+		imgSpec, err := image.ParseRef(o.Config.Mirror.Platform.Release)
+		if err != nil {
+			return []v2alpha1.CopyImageSchema{}, fmt.Errorf("%w", err)
```

Comment:
This defeats the whole purpose of the linter message about adding context to the error. Returning `fmt.Errorf("%w", err)` is the same as just returning the `err` itself, we're not adding any helpful context.

Reply:
Yes we are, its returning the ParseRef formatted error message, why do I need to add more junk to the message  this is just to appease the linter

Reply:
If you just want to appease the linter, then do 
```
return []v2alpha1.CopyImageSchema{}, err //nolint:ireturn // we don't need more context for this
```
`fmt.Errorf("%w", err)` is a pointless wrap.

Reply:
I'm trying to improve this linter message in https://github.com/openshift/oc-mirror/pull/1104